File: CartoMap.Objective-C/CartoMap.Objective-C/CountriesVisController.m

#import "BaseVisController.h"

@interface CountriesVisController : BaseVisController

@end

@implementation CountriesVisController


-(void) viewDidLoad
{
    [self updateVis:@"http://documentation.cartodb.com/api/v2/viz/2b13c956-e7c1-11e2-806b-5404a6a683d5/viz.json"];
}

@end

File: CartoMap.Swift/CartoMap.Swift/CountriesVisController.swift

class CountriesVisController : VisBaseController
{
    override func getName() -> String { return "Countries VisJson"; }
    
    override func getDescription() -> String { return "Displays countries in different colors"; }
    
    var url = "http://documentation.cartodb.com/api/v2/viz/2b13c956-e7c1-11e2-806b-5404a6a683d5/viz.json";
    
    override func viewDidLoad() {
        
        super.viewDidLoad();
        
        mapView?.updateVis(url);
    }
}